web/server/vue-cli/package.json

```diff
@@ -27,21 +27,21 @@
   },
   "dependencies": {
     "@mdi/font": "^5.9.55",
-    "codechecker-api": "file:../../api/js/codechecker-api-node/dist/codechecker-api-6.39.0.tgz",
     "chart.js": "^2.9.4",
     "chartjs-plugin-datalabels": "^0.7.0",
+    "codechecker-api": "file:../../api/js/codechecker-api-node/dist/codechecker-api-6.40.0.tgz",
```

Comment:
@csordasmarton There are still few changes in package.json and pakage-lock.json. Would you want me to hard code the package.json with one if the current source(master) directory?